src/test/resources/jsonld/HT018782520

```diff
@@ -108,15 +115,8 @@
       },
       "type" : [ "Person" ]
     } ],
-    "label" : "Nikolaus (270-340), Heiliger",
+    "label" : "Nikolaus, Heiliger",
```

Comment:
interesting didin't know that Nikolaus is long dead

Reply:
Fun fact: From what I remember he originated from the area which would be Turkey today.